File: src/main/java/net/Chidoziealways/everythingjapanese/worldgen/tree/ModTreeGrowers.java
package net.Chidoziealways.everythingjapanese.worldgen.tree;

import net.Chidoziealways.everythingjapanese.EverythingJapanese;
import net.Chidoziealways.everythingjapanese.worldgen.ModConfiguredFeatures;
import net.minecraft.world.level.block.grower.TreeGrower;

import java.util.Optional;

public class ModTreeGrowers {
    public static final TreeGrower HINOKI = new TreeGrower(EverythingJapanese.MOD_ID + ":hinoki",
            Optional.empty(), Optional.of(ModConfiguredFeatures.HINOKI_KEY), Optional.empty()
    );
}

File: src/main/kotlin/net/Chidoziealways/everythingjapanese/worldgen/tree/ModTreeGrowers.kt
package net.Chidoziealways.everythingjapanese.worldgen.tree

import net.Chidoziealways.everythingjapanese.JAPANESE_MOD_ID
import net.Chidoziealways.everythingjapanese.worldgen.JModConfiguredFeatures
import net.minecraft.world.level.block.grower.TreeGrower
import java.util.*

object ModTreeGrowers {
    val HINOKI: TreeGrower = TreeGrower(
        "$JAPANESE_MOD_ID:hinoki",
        Optional.empty(),
        Optional.of(JModConfiguredFeatures.HINOKI_KEY),
        Optional.empty()
    )
}
